### ISSUE TITLE
Bad chemical formula display in RTL

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/molecules-and-light#142]
![screen shot 2017-04-05 at 4 38 59 pm](https://cloud.githubusercontent.com/assets/480878/24730032/7fad6722-1a1e-11e7-88c2-d8a908aedba6.png)

SubSupText fully assumes LTR direction and scrambles the output (and the string doesn't have the proper LTR marks around the chemical formula).

Using RichText as a drop-in replacement of SubSupText (and adding the LTR/POP bidirectional marks, e.g. `"value": "{0} \u202a({1})\u202c",`), we get:

![screen shot 2017-04-05 at 4 48 15 pm](https://cloud.githubusercontent.com/assets/480878/24730235/b2fb6b50-1a1f-11e7-8151-c50c34d3a42b.png)

@jbphet, could we make this string change through the rosetta system or a commit on babel?

**Comment by jbphet** [comment]
@jonathanolson and I discussed this over Skype, and decided that this could be addressed through a maintenance release to change from `SubSupText` to `RichText` followed by and update to the affected translations using Rosetta.

**Comment by jessegreenberg** [comment]
This sim will soon be redeployed with keyboard navigation.  @jonathanolson @jbphet is this an issue that should be addressed now? If so, let me know if I can help. I am not sure if recent chipper 2.0 work complicates this issue.

**Comment by jbphet** [comment]
@jessegreenberg - yes, I'd like to address it while we have the opportunity.  Is the deployment planned off of master?

**Comment by jessegreenberg** [comment]
@jbphet yes it will be off of master, but possibly with a stable chipper SHA that is before some chipper 2.0 work.
From nov 16: `6f8b4c139ef54b626df1ac052874826d4b646cc0`

**Comment by jessegreenberg** [comment]
@jbphet this is the last issue before this sim is ready for an RC with keyboard navigation, do you have a time estimate for this issue?

**Comment by jbphet** [comment]
@jessegreenberg and I discussed this, and I'll see if I can take care of it quickly today and, if I can't, we will move forward with the release and fix it later.

**Comment by jbphet** [comment]
I looked at it for a while, and my current understanding is that the changes that are still needed are in babel, not the sim code.  I spoke with @jonathanolson, and he mostly agreed, but suggested that I test it.  I tried to do this by moving the Arabic translation of MAL to the "tests" branch of babel and then testing the strings, but this didn't work.  There are a number of things that could have caused this test to fail, but I don't have time to fully investigate right now, so I've advised @jessegreenberg to move forward with the release and I will have to look at this later.

**Comment by phet-steele** [comment]
The above commit successfully forces chemical formulae to be LTR while the rest of the strings stay RTL. Tested on master using `locale=ar` and `locale=fa`. This is fa:

<img width="437" alt="screen shot 2018-02-21 at 12 15 30 pm" src="https://user-images.githubusercontent.com/5863899/36500280-1d9972a2-1701-11e8-8577-e430dbd860ae.png">

**Comment by jbphet** [comment]
@ariel-phet - does this warrant a maintenance release?  The currently published version of MAL for the ar locale is messed up.

**Comment by jbphet** [comment]
Assigning to @ariel-phet to answer question above.

**Comment by ariel-phet** [comment]
@jbphet @jessegreenberg 

I think this might be a good candidate to do a new RC on instead of a maintenance release. I would like to get sims on chipper 2.0 when possible, and this seems like a fairly easy one to test, and QA is not backlogged at the moment, so we have a good window. 

@jessegreenberg does it seem to reasonable to republish off master? If so move forward, if not please reassign me for discussion.

**Comment by jessegreenberg** [comment]
@ariel-phet The only thing holding up a deployment out of master is work for https://github.com/phetsims/molecules-and-light/issues/179, adding methane to this sim. The behavior of methane from the java version was added, but some work needs to go into updating the look of the vibration behavior.

How would you like to proceed? Would you like to finish the work for methane first or remove it and redeploy immediately?

**Comment by jessegreenberg** [comment]
After reviewing I would  expect it to take 4-5 hours to add the requested 3D looking methane to the sim.

**Comment by ariel-phet** [comment]
@jessegreenberg I would like to finish off the methane work.

**Comment by jessegreenberg** [comment]
Sounds good, then this is on hold until #179 is complete. Discussed with @mbarlow12 and removing my assignment.
